appserver/payara-appserver-modules/microprofile/opentracing-adapter/src/main/java/fish/payara/opentracing/OpenTracingService.java

```diff
@@ -94,16 +100,27 @@ public void event(Event<?> event) {
     public synchronized Tracer getTracer(String applicationName) {
```

Comment:
I'd like if we take the change and remove the `synchronized` and use `tracers.computeIfAbsent` instead extracting the initialisation function to a static function in the class. This would both make more clear what the sections of the method do and get rid of some thread contention.